assets/js/atomic/utils/render-parent-block.tsx

```diff
@@ -137,7 +138,7 @@ const renderInnerBlocks = ( {
 		const { blockName = '', ...componentProps } = {
 			key: `${ block }_${ depth }_${ index }`,
 			...( element instanceof HTMLElement ? element.dataset : {} ),
-			className: element.className || '',
+			className: element.className || undefined,
```

Comment:
Having `''` caused `class` to be appended anyway on elements, this caused some elements like `<b>` to have an empty class attribute.